rust/src/ffi/any.rs

```diff
@@ -571,6 +571,33 @@ where
     }
 }
 
+impl<
+        DI: 'static + Domain + Send + Sync,
+        DO: 'static + Domain + Send + Sync,
+        MI: 'static + Metric + Send + Sync,
+        MO: 'static + Metric + Send + Sync,
+    > Transformation<DI, DO, MI, MO>
+where
+    DI::Carrier: 'static + Send + Sync,
+    DO::Carrier: 'static + Send + Sync,
+    MI::Distance: 'static + Send + Sync,
+    MO::Distance: 'static + Send + Sync,
+    (DI, MI): MetricSpace,
+    (DO, MO): MetricSpace,
+{
+    pub fn into_any2(self) -> AnyTransformation {
```

Comment:
What relationship does this have to `into_any`? It seems to be unreferenced -- Is it going to replace the older one?

Reply:
Sorry, that was a fragment of a prior implementation.